src/main/java/com/revlearn/team1/model/Course.java

```diff
@@ -0,0 +1,62 @@
+package com.revlearn.team1.model;
+
+import com.revlearn.team1.enums.AttendanceMethod;
+import jakarta.persistence.*;
+import lombok.Data;
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.UpdateTimestamp;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.util.HashSet;
+import java.util.Set;
+
+@Entity
+@Data
+public class Course {
+
+    @ManyToMany
+    @JoinTable(
+            name = "course_educators",
+            joinColumns = @JoinColumn(name = "course_id"),
+            inverseJoinColumns = @JoinColumn(name = "educator_id")
+    )
+    private final Set<User> educators = new HashSet<>();
```

Comment:
my linter is telling me you need to remove the final fields here: 'if you need to use Lombok’s @Data, remove final from the fields. final fields cannot be modified after initialization, which is incompatible with the typical setter behavior.'

Reply:
speicifically from educators, students, DiscussionPosts